.travis_setup.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -eu
+
+# Install MySQL 5.7 if DB=mysql57
+if [[ -n ${DB-} && x$DB =~ mysql57 ]]; then
+  sudo bash .travis_mysql57.sh
+fi
+
+# Install MariaDB if DB=mariadb
+if [[ -n ${DB-} && x$DB =~ xmariadb ]]; then
+  sudo bash .travis_mariadb.sh "$DB"
+fi
+
+# Install MySQL if OS=darwin
+if [[ x$OSTYPE =~ ^xdarwin ]]; then
+  brew install mysql
+  mysql.server start
+else
+  sudo bash .travis_ssl.sh
+  sudo service mysql restart
+fi
```

Comment:
These should be in separate if blocks. The issue is simply that I couldn't get SSL working on OS X in Travis; in this case I don't want to give the impression that these are mutually exclusive.